scripts/sudoku-grid.js

```diff
@@ -9,6 +9,7 @@ class SudokuGrid {
     };
   }
 
+  //creating the main grid and internal grid lines.
```

Comment:
Comments by convention have a space after the two forward slashes i.e. `// XXXX` not `//XXXX`